<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintaincePortsDataPage</class>
 <widget class="QWidget" name="MaintaincePortsDataPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
		background-color:black;
}
QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
		border-radius:8px;
background-color: rgb(0, 128, 128);
	border-top: 2px solid white;	
	border-left: 2px solid white;
}</string>
  </property>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>621</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>640</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>640</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>640</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>640</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>640</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num6">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>640</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num7">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>640</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num8">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>640</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num9">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>640</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Num0">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>640</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NumA">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>610</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NumC">
   <property name="geometry">
    <rect>
     <x>609</x>
     <y>610</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NumE">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>610</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NumD">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>610</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NumF">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>610</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NumB">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>610</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>590</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>590</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="PortAddressLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
color:rgb(0,128,128);
font:28px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0xFF</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:20px &quot;微软雅黑&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>端口号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
